libraries/aws_ec2_security_groups.rb

```diff
@@ -25,6 +25,10 @@ def access_key_data
     @table
   end
 
+  def count
```

Comment:
I do really like the idea of exposing a `count` property directly rather than via `entries`.  This could be pushed up to a mixin for plural resources one day.

Reply:
Resolved in last push